packages/yelp_xsl.rb

```diff
@@ -17,7 +17,7 @@ class Yelp_xsl < Autotools
      x86_64: '03e70d4b552cb539c1799f1709bf22c20ddcb281b508c462aa146db023f9801c'
   })
 
-  depends_on 'itstool'
+  depends_on 'py3_itstool'
```

Comment:
Rename in `tools/packages.yaml` too?

Reply:
Will that mess up how your update checks script works?

Reply:
Well, the script looks for the url but the name could be anything.

Reply:
Will adjust that then...